packages/http/tools/http-benchmarks/package.json

```diff
@@ -52,11 +52,11 @@
     "url": "git+https://github.com/inversify/monorepo.git"
   },
   "scripts": {
-    "benchmarks:run:cjs": "./bin/run-cjs.cjs",
-    "benchmarks:run:esm": "./bin/run-esm.mjs",
+    "benchmarks:run:cjs": "BUILD_TARGET=cjs ./bin/run-cjs.cjs",
+    "benchmarks:run:esm": "BUILD_TARGET=esm ./bin/run-esm.mjs",
     "build": "pnpm run build:cjs && pnpm run build:esm",
     "build:cjs": "tsc --build tsconfig.cjs.json && pnpm exec foundation-ts-package-cjs ./lib/cjs",
-    "build:esm": "rollup -c ./rollup.config.mjs && pnpm exec foundation-ts-package-esm ./lib/esm",
+    "build:esm": "tsc --build tsconfig.esm.json && pnpm exec foundation-ts-package-esm ./lib/esm",
```

Comment:
Let's remove ESM build for now. Remove package rollup config as well please :slightly_smiling_face: